reduxApp2/dev/js/reducers/reducer-users.js

```diff
@@ -0,0 +1,54 @@
+export default function (state = {}, action) {
+
+  const validateEmail = (email) => {
+    var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+    return re.test(email);
+  }
+
+
+  const validatePhoneNumber = (inputtxt) => {
+    var pnumber = /^\(?([0-9]{3})\)?[-. ]?([0-9]{3})[-. ]?([0-9]{4})$/;
+    if (inputtxt.match(pnumber)) return true;
+    else return false;
+  }
+
+
+  switch (action.type) {
+    case 'ADD_USER':
+      console.log("entered")
+      var {firstName, lastName, email, address, phoneNumber} = action.payload;
+      var valid = {firstName:null, lastName: null, 
```

Comment:
@VARTIKASINGH1234 Please indent the valid object. (one by one line)

Reply:
indented the valid object one by one line